storage/innobase/handler/ha_innodb.cc

```diff
@@ -2163,9 +2163,6 @@ convert_error_code_to_mysql(
 	case DB_TEMP_FILE_WRITE_FAIL:
 		/* This error can happen during
 		copy_data_between_tables() or bulk insert operation */
-		innodb_transaction_abort(thd,
-					 innobase_rollback_on_timeout,
-					 error);
 		my_error(ER_GET_ERRMSG, MYF(0),
                          DB_TEMP_FILE_WRITE_FAIL,
                          ut_strerr(DB_TEMP_FILE_WRITE_FAIL),
```

Comment:
I don’t quite understand this change. When I ran the test case on unmodified db188083c38a9aeba146c9d591e3ed12cce0b200 this code was not being executed.